app/controllers/mentor/analyses_controller.rb

```diff
@@ -3,10 +3,15 @@ def index
     @tracks = Track.where(id: IterationAnalysis.joins(iteration: {person_solution: :exercise}).where('iterations.solution_type': "Solution").select("exercises.track_id"))
 
     @track_id = params[:track_id]
+    @ops_status = params[:ops_status]
+    @analysis_status = params[:analysis_status]
+
     @analyses = IterationAnalysis.includes(iteration: {solution: {exercise: :track}}).
                                   order('id DESC')
 
     @analyses = @analyses.joins(iteration: {person_solution: :exercise}).where('exercises.track_id': @track_id) if @track_id.to_i > 0
+    @analyses = @analyses.where(analysis_status: @ops_status) if @ops_status.present?
```

Comment:
I really do like Ruby's postfix if-statement for these types of filtering things. So elegant!